DOCSUP-13875: Document the replication_wait_for_inactive_replica_timeout setting (#28464)
ClickHouse/ClickHouse: docs/en/sql-reference/statements/optimize.md

The `OPTMIZE` query is supported for [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) family, the [MaterializedView](../../engines/table-engines/special/materializedview.md) and the [Buffer](../../engines/table-engines/special/buffer.md) engines. Other table engines aren’t supported.

> When `OPTIMIZE` is used with the [ReplicatedMergeTree](../../engines/table-engines/mergetree-family/replication.md) family of table engines, ClickHouse creates a task for merging and waits for execution on all nodes (if the [replication_alter_partitions_sync](../../operations/settings/settings.md#replication-alter-partitions-sync) setting is enabled).

Reviewer: It waits for all replicas if `replication_alter_partitions_sync` is set to 2 (and waits for current replica if it is set to 1)

-   If you specify `FINAL`, optimization is performed even when all the data is already in one part. Also merge is forced even if concurrent merges are performed.
-   If you specify `DEDUPLICATE`, then completely identical rows (unless by-clause is specified) will be deduplicated (all columns are compared), it makes sense only for the MergeTree engine.

> You can to specify how long (in seconds) to wait for inactive replicas to execute `OPTIMIZE` queries by the [replication_wait_for_inactive_replica_timeout seconds](../../operations/settings/settings.md#replication-wait-for-inactive-replica-timeout) setting.

Reviewer 2: ```suggestion
You can specify how long (in seconds) to wait for inactive replicas to execute `OPTIMIZE` queries by the [replication_wait_for_inactive_replica_timeout seconds](../../operations/settings/settings.md#replication-wait-for-inactive-replica-timeout) setting.
```

!!! warning "Warning"

>     If the [replication_alter_partitions_sync](../../operations/settings/settings.md#replication-alter-partitions-sync) setting is `2` and some replicas are not active for more than the time specified by the [replication_wait_for_inactive_replica_timeout seconds](../../operations/settings/settings.md#replication-wait-for-inactive-replica-timeout) setting, then an exception `UNFINISHED` will be thrown.

Reviewer 2: ```suggestion
    If the [replication_alter_partitions_sync](../../operations/settings/settings.md#replication-alter-partitions-sync) is set to `2` and some replicas are not active for more than the time, specified by the [replication_wait_for_inactive_replica_timeout seconds](../../operations/settings/settings.md#replication-wait-for-inactive-replica-timeout) setting, then an exception `UNFINISHED` is thrown.
```
